Monika After Story/game/script-topics.rpy

```diff
@@ -12717,6 +12771,55 @@ label monika_unknown:
+    m 4euc "You got to make sure the weather is clear too! Since it's something that happen in the sky, clouds can get in the way..."
+    m 4hksdlb "Not to mention they are pretty rare in the first place..."
+    m 1rksdla "..."
+    m 1hksdlb "Man, the more i think about it, the more complicated it sounds!"
+    m 1tku "But then again, that would make it all the more rewarding when you finally succeed in seeing one."
+    m 1ekbfa "Maybe one day we will get to see one together [player]..."
+    m 3ekbsa "That would be really romantic too, you know?" 
+    m 1dkbsa "Just imagine the two of us..."
+    m 1dkbsa "Laying on a soft mastress of snow, holding hands..."
+    m 1dubsu "Looking up at these dazzling beautiful lights in the sky dancing just for us..."
+    m 1dubsu "Listening to each other's gentle breathing... the freshness of the night air filling our lungs..."
+    m 5hubfu "That would be an experience to remember, don't you think [player]?"
+    m 5eubla "I'll be eagerly waiting for the day we can turn that into a reality."
+    return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_three_musketeers",category=['literature'],prompt="The Three Musketeers",random=True))
+
+label monika_three_musketeers:
+    m 3eub "[player], do you know what a musketeer is?"
+    m 3hub "If you do, it's probably from that book, 'The Three Musketeers'."
+    m 4eub "If you've not heard of it, it's a historical adventure novel from the 19th century, it's considered a classic in literature."
+    m 2rksdla "It's kind of funny you know... the way we picture a whole group of people who actually existed is completely based on a single source."
+    m 2dkc "And from what i've heard, that source is not very historicaly accurate."
+    m 3eub "Nowadays, when you think of a musketeer, most people tend to picture a swashbuckling heroic figure... someone standing up for the weak and the oppressed."
+    m 3eub "Throwing witty quips while fighting in fencing duels... just like the protagonists of the book do."
+    m 4eub "In reality, these guys were a branch of the military; they served either as royal bodyguards, or an elite fighting company during wars."
+    m 4hksdlb "Not much to do with the kind of adventures depicted in the book."
+    m 4wuo "Even their signature fencing sword is a myth. A real life musketeer would favor... a musket! {w=0.5}It's even in their name, musket-eers!"
+    m 1hua "You can really see where the author came from when he said 'You can rape history, so long as you give her a beautiful child'."
+    m 1eua "And it's true it was a pretty entertaining read... "
+    m 3ekb "Yet, i think it's important to keep in mind that for every fiction, the reality behind it may be pretty different from what is depicted."
+    m 1huu "And i don't mean that just for musketeers!"
+    m 2eub "Whether it's pirates, ninjas, or even contemporary things, like policeman, or filmmaker, this applies too!"
```

Comment:
I think "filmmaker" is pluralized with an "s".

Reply:
Pluralized filmmaker and policeman for consistency with the beginning of the sentence.